guides/common/modules/proc_configuring-smart-proxy-for-host-registration-and-provisioning.adoc

```diff
@@ -0,0 +1,57 @@
+[id="configuring-{smart-proxy-context}-for-host-registration-and-provisioning_{context}"]
+= Configuring {SmartProxy} for Host Registration and Provisioning
+
+Use this procedure to configure {SmartProxy} so that you can register and provision hosts using your {SmartProxyServer} instead of your {ProjectServer}.
+
+.Procedure
+. Enable the Registration and Templates features on your {SmartProxyServer} and set the registration and template URLs:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario-smartproxy} \
+--foreman-proxy-registration true \
+--foreman-proxy-registration-url "https://_{smartproxy-example-com}_:{smartproxy_port}" \
```

Comment:
I'm wondering if this is needed. It is optional and if not specified then the same URL as the Smart Proxy is used. By default this should not be needed, but there are edge cases. Most notably when the Foreman server connects to a different URL than the clients do (called mutli-homing). I think we should not assume that's the case and leave this option out:
```suggestion
```